README.md

```diff
@@ -62,6 +62,29 @@ In order to start using `colourista` in your project, you will need to set it up
    main :: IO ()
    main = successMessage "All set up!"
    ```
+### Usage with Stack
+
+`colourista` is not available on Stackage yet, but fear not! You can still use
```

Comment:
Actually `colourista` is available on Stackage! See here: https://www.stackage.org/nightly-2020-03-14/package/colourista-0.0.0.0
It is currently in the `nightly` resolver, however, I guess it will be included in the following `lts`, which I hope will work with GHC-8.8.3.

Reply:
Ahh yes - so what I have done here is erroneously assumed that `not in my current resolver` === `not in Stackage`. Have corrected this now, although really now it's just telling people how to use Stack, and this might not be the place for that.